include/rocksdb/cache.h

```diff
@@ -110,15 +115,21 @@ struct LRUCacheOptions {
                   bool _use_adaptive_mutex = kDefaultToAdaptiveMutex,
                   CacheMetadataChargePolicy _metadata_charge_policy =
                       kDefaultCacheMetadataChargePolicy,
-                  double _low_pri_pool_ratio = 0.5)
+                  double _low_pri_pool_ratio = kDefaultLowPriortyPoolRatio)
       : capacity(_capacity),
         num_shard_bits(_num_shard_bits),
         strict_capacity_limit(_strict_capacity_limit),
         high_pri_pool_ratio(_high_pri_pool_ratio),
-        low_pri_pool_ratio(_low_pri_pool_ratio),
+        low_pri_pool_ratio(_low_pri_pool_ratio == kDefaultLowPriortyPoolRatio
+                               ? 1.0 - _high_pri_pool_ratio
+                               : _low_pri_pool_ratio),
```

Comment:
I feel it would be nicer to deal with this inside `LRUCache`. Also, we could treat all out-of-range (< 0 or > 1) values as "use 1 - high pro pool ratio" (or alternatively, as "no limit on the size of the low-priority pool" as we discussed)